src/main/webapp/resources/js/addOrder.js

```diff
@@ -11,12 +11,41 @@ function activateItem() {
 	var description = listItems[3].value;
 	var contact = listItems[4].value;
 
-	var http = new XMLHttpRequest();
+	var request = new XMLHttpRequest();
 	var url = "/offers";
 	var params = '{"name":"' + name + '","place":"' + place + '","date":"' + date + '","contact":"' + contact + '","description":"' + description + '"}'
-	http.open("POST", url, true);
-	http.setRequestHeader("Content-type", "application/json");
-	http.send(params);
+	request.open("POST", url, true);
+	request.setRequestHeader("Content-type", "application/json");
+    request.onreadystatechange = function() {//Call a function when the state changes.
+    	if(request.readyState == 4 && request.status == 200) {
+    		displayCurrentOffer();
+    	}
+    }
+	request.send(params);
+}
+
+function displayCurrentOffer() {
+    var request = new XMLHttpRequest();
+    var url = "/offers";
+    request.open("GET", url, true);
+
+    request.onload = function() {
+        var offersList = JSON.parse(request.responseText);
+        maxOfferId = offersList.length;
+        displaySingleOffer(maxOfferId);
+    }
+    request.send();
+}
+
```

Comment:
I had some troubles with names of these methods. The method displayCurrentOffer() does two things: first it retrieves id of last-added order and then it calls methods displaySingleOffer() that retrieves certain entry from backend and finally calls method listOffer, which actually 'draw' the offer. So I think we can't name it like getSingleOffer or getOffer since getting offer id is not the only responsibility of this method. But we can consider sth like getAndDisplaySingleOffer(), what do you think?